static/app/utils/replays/playback/hooks/useReplayPlayWhenVisible.tsx

```diff
@@ -1,24 +0,0 @@
-import {useEffect} from 'react';
```

Comment:
same here -- what was this file used for? looks like it got deleted

Reply:
It's not used yet. There's no play/pause for hydration errors so it's kind of an extra unused file at this point. It's related to the larger replayContext.tsx refactor idea, so it'll come back later.